pkg/minikube/problem/err_map.go

```diff
@@ -36,6 +36,12 @@ var vmProblems = map[string]match{
 		URL:    "https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver",
 		Issues: []int{1926, 4206},
 	},
+	"OLD_HYPERKIT_VERSION": {
+		Regexp: re(`Add warn about old driver version (hyperkit, kvm)`),
```

Comment:
Which message is this regexp matching? Do you have an example you a show of where this message is displayed to the user?